tests/eosLightClient/test-verifier-in-EOS.ts

```diff
@@ -0,0 +1,152 @@
+    --expectedHeaderRootPath=${updatePath}`;
+
+    console.info(
+      `Parsing expected new header \n  ╰─➤ ${getExpectedHeaderCommand}`,
+    );
+    const expectedHeaderExec = exec(getExpectedHeaderCommand);
+    const expectedHeader = (await expectedHeaderExec).stdout
+      .toString()
+      .replace(/\s/g, '');
+    console.info(`Parsed expected new header: \n  ╰─➤ [${expectedHeader}]`);
+    await sleep(10000);
+
+    const queryCommand =
+      'cleos push action dendreth printheader "[]" -p dendreth@active';
+    const queryRes = await exec(queryCommand);
+    var result = (await queryRes).stdout.replace(/\s/g, '').substring(36);
+    console.info('Result of query:', result);
+
+    expect(result).toEqual(expectedHeader);
+  }, 30000);
+
+  test('Check "Verifier" after 33 update', async () => {
+    console.info('Verifier after 33 update');
+
+    var updatePath;
+    var counter = 2;
+    for (var proofFilePath of updateFiles.slice(1, 33)) {
+      updatePath = replaceInTextProof(proofFilePath);
+      const updateNumber = updatePath.substring(
+        updatePath.indexOf('update_') + 7,
+      );
+
+      const parseUpdateDataCommand = `${parseDataTool} updateDataEOS \
+      --proofPathEOS=${pathToVerifyUtils}proof_${updateNumber} \
+      --updatePathEOS=${pathToVerifyUtils}update_${updateNumber}`;
+      console.info(
+        `Parsing data for update ` +
+          counter +
+          ` : \n ➤ ${parseUpdateDataCommand}`,
+      );
+      const updateDataExec = exec(parseUpdateDataCommand);
+      const updateData = (await updateDataExec).stdout.replace(/\s/g, '');
+      console.info('update' + counter + 'with data:', updateData);
+      const updateCommand =
+        'cleos push action dendreth update ' +
+        updateData +
+        ' -p dendreth@active';
+      await exec(updateCommand);
+      counter++;
+    }
+
+    //What is the expected result of the query below
+    const getExpectedHeaderCommand = `${parseDataTool} expectedHeaderRootPath \
+    --expectedHeaderRootPath=${updatePath}`;
+
+    console.info(
+      `Parsing expected new header \n  ╰─➤ ${getExpectedHeaderCommand}`,
+    );
+    const expectedHeaderExec = exec(getExpectedHeaderCommand);
+    const expectedHeader = (await expectedHeaderExec).stdout
+      .toString()
+      .replace(/\s/g, '');
+    console.info(`Parsed expected new header: \n  ╰─➤ [${expectedHeader}]`);
+    await sleep(10000);
+
+    const queryCommand =
+      'cleos push action dendreth printheader "[]" -p dendreth@active';
+    const queryRes = await exec(queryCommand);
+    var result = (await queryRes).stdout.replace(/\s/g, '').substring(36);
+    console.info('Result of query:', result);
+
+    const queryCommandall =
+      'cleos push action dendreth printheaders "[]" -p dendreth@active';
+    const queryResall = await exec(queryCommandall);
+    var resultall = (await queryResall).stdout.replace(/\s/g, '').substring(37);
```

Comment:
You can add `expect` statement for this too. Something like that:
  `expect(resultall).toEqual(expectedHeaders);`

Reply:
This could be added in the future together with queries for the last FinalizedHeader and the last ExecutionStateRoot